eclair-core/src/main/scala/fr/acinq/eclair/wire/internal/channel/version1/ChannelCodecs1.scala

```diff
@@ -62,9 +62,11 @@ private[channel] object ChannelCodecs1 {
         ("toSelfDelay" | cltvExpiryDelta) ::
         ("maxAcceptedHtlcs" | uint16) ::
         ("isInitiator" | bool8) ::
-        ("defaultFinalScriptPubKey" | lengthDelimited(bytes)) ::
+        ("upfrontShutdownScript_opt" | lengthDelimited(bytes).map[Option[ByteVector]](b => if (b.isEmpty) None else Some(b)).decodeOnly) ::
```

Comment:
```suggestion
        ("upfrontShutdownScript_opt" | lengthDelimited(bytes).map(Option(_)) ::
```

Reply:
done  in https://github.com/ACINQ/eclair/pull/2565/commits/e50a30eb4fbfb35dd74fd521bd2d9b1255315207